packages/ui/src/components/Select/style.less

```diff
@@ -0,0 +1,142 @@
+@import "antd/lib/select/style/index.less";
+@import (reference) "@open-condo/ui/src/tokens/variables.less";
+@import (reference) "@open-condo/ui/src/components/style/mixins/typography";
+@import "./mixins";
+
+@select-border-color: @condo-global-color-gray-5;
+
+.condo-select {
+  align-items: center;
+
+  // Display modes
+  &-fit-content {
+    display: inline-flex;
+    width: min-content;
+  }
+  &-fill-parent {
+    display: flex;
+    width: 100%;
+  }
+
+  // Select types
+  .condo-select-type("danger", @condo-global-color-red-5);
+  .condo-select-type("success", @condo-global-color-green-5);
+  .condo-select-type("warning", @condo-global-color-orange-5);
+  .condo-select-type("info", @condo-global-color-blue-5);
+  .condo-select-type("secondary", @condo-global-color-gray-7);
+
+  &-danger > &-arrow, &-warning > &-arrow, &-info > &-arrow, &-secondary > &-arrow, &-success > &-arrow {
+    color: @condo-global-color-white;
+  }
+
+  &-danger &-selection-item > *,
+  &-warning &-selection-item > *,
+  &-info &-selection-item > *,
+  &-secondary &-selection-item > *,
+  &-success &-selection-item > * {
+    color: @condo-global-color-white;
+  }
+
+  &-arrow {
+    position: absolute;
+    margin: 0;
+    top: 18px;
```

Comment:
Same about % 4, is it ok?

Reply:
I guess this is related to icon container size